cmd/soroban-cli/build.rs

```diff
@@ -1,3 +1,124 @@
 fn main() {
     crate_git_revision::init();
+    build_helper::set_example_contracts();
+}
+
+mod build_helper {
```

Comment:
I wonder if this should be pulled out into its own file, or if it's okay here for now.

Reply:
Good question - just tested this locally and haven't noticed a compile time compared to `main`. So I'll plan to merge it in as-is, and will keep an eye out in case it becomes an issue.